src/xopen/__init__.py

```diff
@@ -808,7 +850,7 @@ def xopen(  # noqa: C901
     compresslevel is the compression level for writing to gzip, xz and zst files.
     This parameter is ignored for the other compression formats.
     If set to None, a default depending on the format is used:
-    gzip: 6, xz: 6, zstd: 3.
+    gzip: 6, xz: 6, zstd: 3, lz4: 1.
```

Comment:
This doesn’t quite match that `XOPEN_DEFAULT_LZ4_COMPRESSION` is set to zero above.